telmos_main.py

```diff
@@ -210,11 +210,13 @@ def apply_pivot_files(tod_data, cte_data,
                         airport_growth[:,None])
         # These columns do the following
         if j in [0,3,4,7,8]:
-            sw_cte_array[j,:,8] = sw_array[j,:,5]
+            sw_cte_array[j,:,8] = (cte_data[j,:,8] * 
```

Comment:
Previously it was using the data from the base TOD file and applying growth to calculate the forecast CTE file. Although as the TOD files are to a lower precision - 3 decimal places instead of 5 - it was not always accurately recreating the base if the same scenario planning data was used.
This now uses the base CTE file to calculate the forecast version. Not sure why this was not done before.